DemoApp/ios/Podfile

```diff
@@ -39,5 +39,13 @@ target 'DemoApp' do
       :mac_catalyst_enabled => false
     )
     __apply_Xcode_12_5_M1_post_install_workaround(installer)
+
+    # Workaround for Xcode 14.3:
+    # https://github.com/facebook/react-native/issues/34106#issuecomment-1417685116
+    installer.pods_project.targets.each do |target|
+      target.build_configurations.each do |config|
+        config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '12.4'
```

Comment:
why is it 12.4 if ours is 14..

Reply:
This is for the react native pods - it was some of the dependencies that targeted 11.0 but used an API from Apple in 12.4. It's for the DemoApp so I assume our clients also had the same issue and used a similar workaround, let me try change it for a specific pod and see if it works

Reply:
If it's RN released that fine, you might as well keep it this way.